src/scripts/utils/swRegister.es6

```diff
@@ -0,0 +1,5 @@
+if ('serviceWorker' in navigator) {
+	navigator.serviceWorker.register('serviceWorker.js').then((registration) => {
+		console.log(registration)
+	})
```

Comment:
Když SW selže, tak se nic nestane. Je to jen enhancement,

Že je s tím problém zjistíš v developer tools.